values-nl/strings.xml

```diff
@@ -28,27 +28,27 @@
             <string name="pref_header_install">Installatie</string>
             <!--<string name="pref_header_download">Download (noun variant, not used in English)</string>-->
             <string name="new_folder">Nieuwe map</string>
-            <string name="pref_header_gapps_selection">GApps-Selectie</string>
+            <string name="pref_header_gapps_selection">GApps-selectie</string>
             <string name="action_settings">Instellingen</string>
-            <string name="label_wifi_only">Alleen op WiFi</string>
-            <string name="label_download_dir">Downloadmap</string>
+            <string name="label_wifi_only">Alleen via wifi</string>
+            <string name="label_download_dir">Downloadlocatie</string>
             <string name="label_wipe_cache">Cache/Dalvik wissen</string>
             <string name="label_download_md5">Download MD5</string>
             <string name="only_newest_version">Automatisch downloads opschonen</string>
-            <string name="label_download_versionlog">Download VersieLog</string>
-            <string name="label_root_mode">Gebruik Root Rechten</string>
+            <string name="label_download_versionlog">Download versielog</string>
+            <string name="label_root_mode">Gebruik rootrechten</string>
         <!--SUMMARIES-->
-            <string name="summary_wifi_only">Enkel over Wifi downloaden</string>
+            <string name="summary_wifi_only">Enkel via wifi downloaden</string>
             <string name="explanation_newest_version">Verwijder verouderde versie als een nieuwe gelijke variant gedownload is</string>
-            <string name="explanation_download_md5">Download MD5-Checksum om pakketintegriteit te verifiëren</string>
-            <string name="explanation_download_versionlog">Download VersieLog om de pakketinhoud te tonen</string>
-            <string name="summary_root_mode">Geef root toegang voor het gebruik van installatie-modus en de wipe cache/dalvik optie</string>
+            <string name="explanation_download_md5">Download MD5-checksum om pakketintegriteit te verifiëren</string>
+            <string name="explanation_download_versionlog">Download versielog om de pakketinhoud te tonen</string>
+            <string name="summary_root_mode">Geef roottoegang voor het gebruik van de installatiemodus en de optie om de (dalvik)cache te wissen</string>
```

Comment:
Bij root-toegang kunnen we dat wel doen, ja. Ook omdat het een Engels woord betreft.

cache/dalvik: klopt, is helemaal waar.